tests/test/check/data/main.fmf

```diff
@@ -22,3 +22,46 @@
       sudo bash -c "passwd --help &> /root/passwd.log; \
                     ls -alZ /root/passwd.log; \
                     rm -f /root/passwd.log" || /bin/true
+
+/watchdog/ping:
+  require:
+    - /usr/bin/uptime
+
+  test: |
+    set -x
+
+    export
+    uptime
+
+    if [ "$TMT_REBOOT_COUNT" = "1" ]; then exit 0; fi
+
+    # Collect a couple of successful reponse
```

Comment:
```suggestion
    # Collect a couple of successful responses
```

Reply:
Fixed in 8a5ec98ee0dfd6a01582c6837d2f3b3cf9094f73